### ISSUE TITLE
Lobbyid issue - Cannot joinPracticeLobby or configPracticeLobby 

### ISSUE THREAD

**Issue by SwaroopH** [RJacksonm1/node-dota2#39]
Hi,

I tried using the generated lobby_id to configure or re-join (after leaving) but it always throws a DOTA_JOIN_RESULT_INVALID_LOBBY.

I even created a lobby from another account using the official client, grabbed the lobby id from console but that resulted in the same error – with or without passwords.

Am I missing something?

**Comment by paralin** [comment]
The lobby ID issue is because it detects any cache subscription as a lobby. This is wrong.

**Comment by Maxpain177** [comment]
some problem...
lobby creates normally, but join/config doesn't work!
```
Dota2.joinPracticeLobby("23958121502216556", "password", console.log)
// DOTA_JOIN_RESULT_INVALID_LOBBY { result: 'DOTA_JOIN_RESULT_INVALID_LOBBY' }
```

**Comment by paralin** [comment]
@Maxpain177 Probably the lobby ID you are specifying there is invalid. It changes each time you make a lobby.

**Comment by Maxpain177** [comment]
@paralin, i replaced protobuf module on module, called "node-protobuf" and this works fine.

**Comment by Maxpain177** [comment]
And i pass preserve_int64 argument (true).

**Comment by paralin** [comment]
@Maxpain177 I don't think there's anything wrong with the protobuf module, but I think preserving the int64 sounds interesting. That might be the problem.

**Comment by Maxpain177** [comment]
@paralin, i can make a pull-request today

**Comment by RJacksonm1** [comment]
Should be fixed by #66.
